api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -104,7 +105,13 @@ message Policy {
   // An optional symbolic expression specifying an access control
   // :ref:`condition <arch_overview_condition>`. The condition is combined
   // with the permissions and the principals as a clause with AND semantics.
-  google.api.expr.v1alpha1.Expr condition = 3;
+  oneof expression_specifier {
```

Comment:
Adding a `oneof` around an existing field is a breaking change in the golang proto API.  For now, you will need to add the new field without the `oneof`, but you can add an annotation indicating that in the next major version of the API, these two fields should be promoted into the same `oneof`.  I don't remember offhand how to do that, but @htuch can probably tell you.

Reply:
changed to two separate fields with migrate tag.